experimental/framework/tree-react-api/src/reactSharedTreeView.tsx

```diff
@@ -238,15 +241,15 @@ function useViewRoot<TSchema extends ImplicitFieldSchema>(
 /**
  * React component which handles schematizing trees.
  * This includes displaying errors when the document can not be schematized.
+ * @public
  */
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-function TreeViewComponent<TSchema extends ImplicitFieldSchema>({
+export function TreeViewComponent<TSchema extends ImplicitFieldSchema>({
 	tree,
 	viewComponent: ViewComponent,
 	errorComponent,
 }: TreeViewProps<TSchema> & {
-	tree: ReactTreeDataObject<TSchema>;
-}) {
+	tree: Pick<IReactTreeDataObject<TSchema>, "treeView">;
+}): React.JSX.Element {
```

Comment:
I think you can simplify this to just
```suggestion
}): JSX.Element {
```

Reply:
But that's `@deprecated — Use React.JSX instead of the global JSX namespace.`